tests/unit/store.t

```diff
@@ -8,7 +8,7 @@ use Log::Any::Adapter 'TAP';
 
 use ProductOpener::Store qw/:all/;
 
-is( get_fileid('Do not challenge me!'), 'do-not-challenge-me' );
+is( get_fileid('Do not challenge me!', 0, 'en'), 'do-not-challenge-me' )
```

Comment:
```suggestion
is( get_fileid('Do not challenge me!', 0, 'en'), 'do-not-challenge-me' );
```
It should also work with just one argument now that the rest of the arguments are optional.